addons/website_sale_comparison/static/src/scss/website_sale_comparison.scss

```diff
@@ -1,6 +1,6 @@
 .o_product_feature_panel {
     position: fixed;
-    bottom: 0;
+    bottom: var(--move-cookie-over-modal, 0px) !important;
```

Comment:
wont this change will affect the product snippet already dropped in the website ?

we need to use the `data-vcss='file_name'` concept to address this changes for stable versions.

Reply:
@shsa-odoo , if we dont get a variable, then its default value will be 0px only. So, I do not think we need to write data-vcss for this.

Reply:
Marking it !important will have some effect intially it was not marked important, so if you have tested that and its not impacting, looks good then.